<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GammaWidget</class>
 <widget class="QWidget" name="GammaWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item rowspan="2" row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Red</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="3" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QDoubleSpinBox" name="spnBlue" >
     <property name="maximum" >
      <double>10</double>
     </property>
     <property name="minimum" >
      <double>0.1</double>
     </property>
     <property name="singleStep" >
      <double>0.1</double>
     </property>
     <property name="value" >
      <double>1</double>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="4" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Blue</string>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QDoubleSpinBox" name="spnGreen" >
     <property name="maximum" >
      <double>10</double>
     </property>
     <property name="minimum" >
      <double>0.1</double>
     </property>
     <property name="singleStep" >
      <double>0.1</double>
     </property>
     <property name="value" >
      <double>1</double>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Master</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDoubleSpinBox" name="spnMaster" >
     <property name="maximum" >
      <double>10</double>
     </property>
     <property name="minimum" >
      <double>0.1</double>
     </property>
     <property name="singleStep" >
      <double>0.1</double>
     </property>
     <property name="value" >
      <double>1</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QDoubleSpinBox" name="spnRed" >
     <property name="maximum" >
      <double>10</double>
     </property>
     <property name="minimum" >
      <double>0.1</double>
     </property>
     <property name="singleStep" >
      <double>0.1</double>
     </property>
     <property name="value" >
      <double>1</double>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QCheckBox" name="chkPreview" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>&amp;Preview</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="5" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
